### ISSUE TITLE
25-test_x509.t broke at master on NonStop

### ISSUE THREAD

**Issue by rsbeckerca** [openssl/openssl#24189]
I'm looking for help on figuring out this regression
```
25-test_x509.t .......................... 40/?
Certificate request self-signature ok
subject=CN=b.example.com
../../util/wrap.pl ../../apps/openssl x509 -req -text -key b-key.pem -not_before 20231031000000Z -not_after today -in b-cert.csr -out b-cert.pem => 0
not ok 46
# ------------------------------------------------------------------------------
Certificate request self-signature ok
subject=CN=b.example.com
../../util/wrap.pl ../../apps/openssl x509 -req -text -key b-key.pem -not_before today -days 99 -in b-cert.csr -out b-cert.pem => 0
not ok 48
# ------------------------------------------------------------------------------
Certificate request self-signature ok
subject=CN=b.example.com
../../util/wrap.pl ../../apps/openssl x509 -req -text -key b-key.pem -in b-cert.csr -out b-cert.pem => 0
not ok 50
# ------------------------------------------------------------------------------
25-test_x509.t .......................... Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/51 subtests
        (less 1 skipped subtest: 47 okay)
```

**Comment by nhorman** [comment]
can you bisect to see when this was introduced?

**Comment by rsbeckerca** [comment]
Not sure, but it is fixed at 4e3c1e6206
